python/hsfs/util.py

```diff
@@ -102,48 +103,11 @@ def create_mysql_engine(online_conn, external):
     return sql_alchemy_engine
 
 
-def check_timestamp_format_from_date_string(input_date):
```

Comment:
What happened to this method? are you sure it's not used anywhere else?

Reply:
dateutils parser covers all former date parsing + some and all the unit test for it passed.

Reply:
ok great